lib/internal/Magento/Framework/DB/Adapter/Pdo/Mysql.php

```diff
@@ -1475,10 +1475,10 @@ public function select()
      * Method revrited for handle empty arrays in value param
      *
      * @param string $text The text with a placeholder.
-     * @param mixed $value The value to quote.
-     * @param string $type OPTIONAL SQL datatype
+     * @param array|null|int|string|float|\Zend_Db_Expr|\Zend_Db_Select|\DateTimeInterface $value The value to quote.
```

Comment:
I'd like to use `\Magento\Framework\DB\Sql\Expression` instead of `\Zend_Db_Expr` and `\Magento\Framework\DB\Select` instead of `\Zend_Db_Select` in order to have an ability to replace ZF1 implementation in future.
```suggestion
     * @param array|null|int|string|float|\Magento\Framework\DB\Sql\Expression|\Magento\Framework\DB\Select|\DateTimeInterface $value The value to quote.
```
Also would be good to import it, but github UI don't provide ability to do it